src/libs/kata-types/src/config/agent.rs

```diff
@@ -80,6 +80,24 @@ pub struct Agent {
     pub container_pipe_size: u32,
 }
 
+impl std::default::Default for Agent {
+    fn default() -> Self {
+        Self {
+            debug: true,
+            enable_tracing: false,
+            server_port: DEFAULT_AGENT_VSOCK_PORT,
+            log_port: DEFAULT_AGENT_LOG_PORT,
+            dial_timeout_ms: 10,
```

Comment:
What about making the constants here a variable, like
`const AGENT_DIAL_TIMEOUT_MS: u32 = 10;`

Reply:
Done.